ratis-server-api/src/main/java/org/apache/ratis/server/RaftServerConfigKeys.java

```diff
@@ -109,6 +109,18 @@ static SizeInBytes byteLimit(RaftProperties properties) {
     static void setByteLimit(RaftProperties properties, SizeInBytes byteLimit) {
       setSizeInBytes(properties::set, BYTE_LIMIT_KEY, byteLimit, requireMin(1L));
     }
+
+    String MEGA_BYTE_LIMIT_KEY = PREFIX + ".megabyte-limit";
```

Comment:
Let's reuse BYTE_LIMIT_KEY.  Otherwise, it is confusing to have two keys.

Reply:
done.